<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="SearchingTest" tests="2" time="0.057" ignored="0" failures="1" timestamp="13 апр 2018 11:00:14 GMT" skipped="1" hostname="Роман-ПК" errors="0">
  <testcase name="CheckSearching" time="0.057" classname="SearchingTest">
    <skipped/>
  </testcase> <!-- CheckSearching -->
  <testcase name="Initialize" time="0.000" classname="SearchingTest">
    <failure type="java.lang.AssertionError" message="expected [РЇРЅРґРµРєСЃ.РњР°СЂРєРµС‚ вЂ” РІС‹Р±РѕСЂ Рё РїРѕРєСѓРїРєР° С‚РѕРІР°СЂРѕРІ РёР· РїСЂРѕРІРµСЂРµРЅРЅС‹С… РёРЅС‚РµСЂРЅРµС‚-РјР°РіР°Р·РёРЅРѕРІ] but found [Яндекс.Маркет — выбор и покупка товаров из проверенных интернет-магазинов]">
      <![CDATA[java.lang.AssertionError: expected [РЇРЅРґРµРєСЃ.РњР°СЂРєРµС‚ вЂ” РІС‹Р±РѕСЂ Рё РїРѕРєСѓРїРєР° С‚РѕРІР°СЂРѕРІ РёР· РїСЂРѕРІРµСЂРµРЅРЅС‹С… РёРЅС‚РµСЂРЅРµС‚-РјР°РіР°Р·РёРЅРѕРІ] but found [Яндекс.Маркет — выбор и покупка товаров из проверенных интернет-магазинов]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at SearchingTest.Initialize(SearchingTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.TestRunner.beforeRun(TestRunner.java:634)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
    </failure>
  </testcase> <!-- Initialize -->
</testsuite> <!-- SearchingTest -->
